Pixelui shouldn't crash if font file not found

Currently, if PixelUI.AddTTFFont is called on a file that doesn't exist, the program crashes with an un-helpful error. PixelUI should be updated to check if the font file exists before trying to add it to imgui.

Fair, that's probably what I'll do with a proper message... the problem currently is that the C layer is handling the file open and basically just explodes unhelpfully if the file doesn't exist.

```
panic: Assertion failed!
File: imgui_draw.cpp, Line 1605

Expression: 0


goroutine 6 [running]:
github.com/inkyblackness/imgui-go.glob..func1({0xc00001202c, 0x1}, {0xc000012040, 0xc0000583e0}, 0x0)
```

--- a/font.go
+++ b/font.go
@@ -1,8 +1,10 @@
 package pixelui
 
 import (
+	"fmt"
 	"image/color"
 	"log"
+	"os"
 	"unsafe"
 
 	"github.com/inkyblackness/imgui-go"
@@ -40,6 +42,9 @@ func (ui *UI) loadDefaultFont() {
 
 // AddTTFFont loads the given font into imgui.
 func (ui *UI) AddTTFFont(path string, size float32) {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		panic(fmt.Sprintf("The font file: %s does not exist", path))
+	}
 	ui.fonts.AddFontFromFileTTF(path, size)
 	ui.loadFont()
 }

